### PR TITLE
Prepare for v0.8

### PR DESCRIPTION
**Pull request by ludwigschwardt**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -3,26 +3,47 @@ MeerKAT Science Data Processor Telescope State
...
 
 .. warning::
 
   **WARNING**: The standard warning about Python pickles applies. Never
-  retrieve data from an untrusted telescope state or connect to one over an
-  untrusted network.
+  retrieve data from an untrusted telstate database with values encoded as
+  pickles, or connect to such a database over an untrusted network.
+  This should only be an issue with older telstates stored in RDB files.
+
```
Comment (bmerry) [inline README.rst]: Document the environment variable that controls whether pickles can be loaded. Disabling pickles should make it safe to connect to untrusted telstates.

File: README.rst [hunk 2]
```diff
@@ -3,26 +3,47 @@ MeerKAT Science Data Processor Telescope State
...
 timestamp that is not allowed to change.
 
-The keys are strings and values are arbitrary Python objects stored as pickles.
-Keys can be accessed using attribute syntax or dict syntax.
+The keys are strings and the values are Python objects serialised via
+MessagePack_. Older versions of the database stored the values as pickles, and
+the package warns the user if that's the case. Keys can be retrieved from the
+telstate object using attribute syntax or dict syntax.
+
+.. _MessagePack: http://www.msgpack.org/
```
Comment (bmerry) [inline README.rst]: Maybe mention that it's been extended to support tuples, complex numbers and numpy arrays.

File: README.rst (hunk 1 above)
Comment (ludwigschwardt) [reply README.rst]: Done.

File: README.rst (hunk 2 above)
Comment (ludwigschwardt) [reply README.rst]: Done.

**Pull request merged**